lib/common.sh

```diff
@@ -733,10 +682,10 @@ common::check_packages () {
             "Run the following and try again:"
     logecho
     for prereq in ${missing[@]}; do
-      if [[ $prereq == "sendgmr" ]] && [[ $distro == "Ubuntu" ]]; then
-        logecho "sudo goobuntu-add-repo $prereq && sudo apt-get update"
+      if [[ -n ${PREREQUISITE_INSTRUCTIONS[$prereq]} ]]; then
+        logecho "# See ${PREREQUISITE_INSTRUCTIONS[$prereq]}"
       fi
-      logecho "sudo $packagemgr install $prereq"
+      logecho "$ sudo $packagemgr install $prereq"
```

Comment:
maybe this should be under an `else`? i.e. only echo this if there are no instructions?